go/libraries/doltcore/sqle/enginetest/dolt_queries_merge.go

```diff
@@ -4045,7 +4045,34 @@ var OldFormatMergeConflictsAndCVsScripts = []queries.ScriptTest{
 	},
 }
 
-var verifyConstraintsSetupScript = []string{
+var verifyConstraintsUniqueViolationsSetupScript = []string{
```

Comment:
Maybe these belong in their own file?

Reply:
totally agree. I was going to move them after review – I just figured the diff would be easier to read before that. I should'a mentioned that in the PR description.